templates/includes/navbar.html

```diff
@@ -0,0 +1,31 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light" style="color: black">
```

Comment:
Code quality issue: Inline styles should be avoided. It's better to move the `color: black` style to a CSS class in your styles.css file and then add this class to the navbar element. This makes your code cleaner and more maintainable.